src/main/views/macro/table.njk

```diff
@@ -20,7 +20,12 @@
     </div>
     {% if changeLink %}
       <div class="column-one-quarter change-link-column">
-        <a href="{{ changeLink | safe }}" class="bold">{{ t('Change') }} <span class="visuallyhidden">{{ label | lower }}</span></a>
+        {% if (label === 'Full name') %}
```

Comment:
There are several such scenarios in both claimant and defendant journey. These needs to be addressed